packages/flutter/test/material/autocomplete_test.dart

```diff
@@ -336,8 +339,8 @@ void main() {
     final Finder listFinder = find.byType(ListView);
     expect(listFinder, findsNothing);
 
-    /// entering `a` returns 9 items(height > `maxOptionsheight`) from the kOptions
-    /// so height gets restricted to `maxOptionsheight =250`
+    /// entering `a` returns 9 items(height > `maxOptionsHeight`) from the kOptions
```

Comment:
Not sure what's being changed here. It looks like the comments in this test should be `//` rather than `///` and sentences (start with a capital letter, end with a period).

Reply:
Sure thing. I was just fixing the `maxOptionsheight` -> `maxOptionsHeight` typo. But your comment is valid. I will update this.